addons/web_tour/static/src/js/tour_manager.js

```diff
@@ -391,10 +392,8 @@ return core.Class.extend(mixins.EventDispatcherMixin, ServicesMixin, {
         //display rainbow at the end of any tour
         if (this.tours[tour_name].rainbowMan && this.running_tour !== tour_name &&
             this.tours[tour_name].current_step === this.tours[tour_name].steps.length) {
-            var $rainbow_message = $('<strong>' +
-                                '<b>Good job!</b>' +
-                                ' You went through all steps of this tour.' +
-                                '</strong>');
+            var $rainbow_message = this.tours[tour_name].rainbowManMessage ||
+                $('<strong><b>Good job!</b> You went through all steps of this tour.</strong>');
```

Comment:
This wasn't translated :thinking: ? Could you make it translatable while you are at it ? Thanks :)

Reply:
Sure